doc/monitoring/metrics_reference.rst

```diff
@@ -6,6 +6,23 @@ Metrics reference
 
 This page provides detailed description of metrics from module ``metrics``.
 
+-------------------------------------------------------------------------------
+General metrics
+-------------------------------------------------------------------------------
+
+General instance information.
+
+* ``tnt_cfg_current_time`` - instance system time in uxix timestamp format
+
+* ``tnt_cfg_listen`` - Tarantool port number
```

Comment:
I suggest to not document metrics we are not collecting https://github.com/tarantool/metrics/pull/43/files